lib/components/map/nearby-view-dot-overlay.tsx

```diff
@@ -33,8 +33,11 @@ const NearbyViewDotOverlay = ({ location }: Props) => {
 
 const mapStateToProps = (state: AppReduxState) => {
   const { highlightedLocation } = state.otp.ui
+  const { nearbyView } = state.otp.config
+
   return {
-    location: highlightedLocation
+    location:
+      nearbyView?.showShadowDotOnMapDrag === false ? null : highlightedLocation
```

Comment:
cmon now

`nearbyView?.showShadowDotOnMapDrag ? highlightedLocation : null`

Reply:
we need it on my default!

Reply:
If the shadow dot should be on by default, maybe hideShadowDotOnMapDrag instead?

Reply:
but then you're dealing with a double negative :(

Reply:
Maybe, but it would tidy up this line, plus I think as a config item the phrasing of `hideShadowDotOnMapDrag` makes it a little more clear how you're impacting the default configuration.

Reply:
We've done this strategy many times before and it keeps things much clearer than getting confused with double negatives